### PR TITLE
[Task]: Minor ministries page fixes

### PR DESCRIPTION
**Pull request by Alexandra-Green**
## What this PR accomplishes

- All content into landmark container
- Changed the id of header search bar so the Search bar and header search bar do not have the same id.
- Remove placeholder text from search bars in search pages
- Search results have the role="status"
- Changed breadcrumb icon and removed current page from breadcrumbs list
- Removed clearfix items in ministries list

## Issue(s) addressed

- DATA-1288
- DATA-1298
- DATA-1297

## What needs review

- Breadcrumbs of current page are omitted across the site
- Content of the Ministry page is in a `role="main"` landmark
- No clearfix items in Ministry page list
- No placeholders in search pages search bars
- Search results on search pages have `role="status"`
- Header search bar submit button does not have the same id as the search bar submit button on Ministries page

### CODE REVIEW

**Comment by KatiRG** [comment]
- [ ]  Breadcrumbs of current page are omitted across the site
- [ ]     Content of the Ministry page is in a role="main" landmark
- [ ]     No clearfix items in Ministry page list
- [ ]     No placeholders in search pages search bars
- [ ]     Search results on search pages have role="status"
- [ ]     Header search bar submit button does not have the same id as the search bar submit button on Ministries page

**Comment by KatiRG** [comment]
There is still a placeholder text in the application header searchbar. Is this intentional?
Screenshot 2023-10-03 at 13 47 51

**Comment by Alexandra-Green** [comment]
@KatiRG yes, removing the header search bar placeholder is a request in ticket 1287, it will also remove the placeholder on the homepage. This PR refers to the search pages specifically, because the same code is being reused for each search page.

File: ckanext/ontario_theme/templates/internal/package/base.html [hunk 1]
```diff
@@ -16,12 +16,7 @@
     {% else %}
       <li>{% link_for _('Datasets'), 'dataset.search' %}</li>
     {% endif %}
-    <li id="dataset-name" dataset-name="{{ pkg.title }}" dataset-name-fr="{{ h.ontario_theme_get_translated_lang(pkg, 'title', 'fr') }}" {{ self.breadcrumb_content_selected() }}>{% link_for dataset|truncate(30), named_route='dataset.read', id=pkg.name %}
```
Comment (KatiRG) [inline ckanext/ontario_theme/templates/internal/package/base.html:19]: I think this removes the dataset name when editing a resource or adding a new resource to an existing dataset

**Comment by KatiRG** [comment]
The dataset name is now back in the breadcrumbs when editing a dataset page, and when editing an existing resource, but gets dropped when adding a new resource:
Screenshot 2023-10-03 at 16 58 46

**Comment by Alexandra-Green** [comment]
> The dataset name is now back in the breadcrumbs when editing a dataset page, and when editing an existing resource, but gets dropped when adding a new resource: 

Added it back in new commit

**Review by KatiRG — CHANGES_REQUESTED** [review]

**Review by KatiRG — APPROVED** [review]
Looks good! Great work Alex!

**Pull request merged**